java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -460,6 +461,11 @@ public CompletableFuture<Long> scard(@NonNull String key) {
         return commandManager.submitNewCommand(SCard, new String[] {key}, this::handleLongResponse);
     }
 
+    @Override
+    public CompletableFuture<Set<String>> sinter(@NonNull String[] keys) {
+        return commandManager.submitNewCommand(SInter, keys, this::handleSetResponse);
```

Comment:
Note: Let's ask about the return value, as redis documentation says that it returns an Array 🤨

Reply:
Tested with redis-cli - the doc is outdated.
RESP2 returns array, RESP3 returns a set
I added data convertion to glide-core which used with RESP2
```
$ redis-cli -3
127.0.0.1:6379> sinter s1
1~ "a"
127.0.0.1:6379> sinter s1
1~ "a"
127.0.0.1:6379> sinter s1 s2
(empty set)
127.0.0.1:6379> smembers s1
1~ "a"
127.0.0.1:6379>
```